pkg/ndctl/ndctl.go

```diff
@@ -62,14 +66,18 @@ func (ctx *Context) GetBuses() []*Bus {
 
 //CreateNamespace create new namespace with given opts
 func (ctx *Context) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error) {
+	var err error
+	var ns *Namespace
 	for _, bus := range ctx.GetBuses() {
 		for _, r := range bus.ActiveRegions() {
-			if ns, err := r.CreateNamespace(opts); err == nil {
+			if ns, err = r.CreateNamespace(opts); err == nil {
 				return ns, nil
+			} else {
+				err = errors.Wrap(err, "")
```

Comment:
And here you add an empty prefix. I don't know exactly how `errors.Wrap` handles that. Probably it simply ignores the string and just adds a stack entry.